var/spack/repos/builtin/packages/mstk/package.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Can you revert the change to the copyright?

Reply:
Done